src/http/client.rs

```diff
@@ -27,6 +27,15 @@ impl HttpClient {
       .connect_timeout(Duration::from_secs(60))
       .user_agent("Tailcall/1.0");
 
+    if let Some(ref http_settings) = server.http {
+      builder = builder
+        .http2_keep_alive_interval(Some(Duration::from_millis(http_settings.keep_alive_interval)))
+        .http2_keep_alive_timeout(Duration::from_millis(http_settings.keep_alive_timeout))
+        .http2_keep_alive_while_idle(http_settings.keep_alive_while_idle)
+        .pool_idle_timeout(Some(Duration::from_millis(http_settings.pool_idle_timeout)))
+        .pool_max_idle_per_host(http_settings.pool_max_idle_per_host)
+    }
```

Comment:
I think this implementation is good for now, we should just add the options that the client is already configured with right above your changes — https://github.com/tailcallhq/tailcall/pull/388/files#diff-949af4887508b0c9ff8c0ba77a47a53fe8d3cc46c940dd8218463517e052b12fR23

That way we build the builder in one go.

Reply:
Got it.  Thanks
Should I support all fields or only the above fields would suffice for now?

Reply:
Just the above fields for now. I think we will have to go thru each one of them to see what makes sense and what doesn't. Let's keep it simple for now.